### DISCUSSION TITLE
Back End

### DISCUSSION THREAD

**Discussion by LoriaLawrenceZ** [ZimbresAguilar/Ozza-Tech#4] [Ideas]
Chat para debater questões relacionas a back-end

**Comment by Paiva8859** [comment]
@LoriaLawrenceZ

**Reply by LoriaLawrenceZ** [reply]
Fala fi, marcou e n escreveu nada

**Reply by Paiva8859** [reply]
Foi tu q se marcou ontem. Quando a gente tava resolvendo os PR. Até falou "meio sem sentido eu me marcar aqui" e enviou kkkk

**Comment by Paiva8859** [comment]
@LoriaLawrenceZ

**Comment by LoriaLawrenceZ** [comment]
@Paiva8859 

Link para a videoaula [PHP + Firebase]:
https://youtu.be/06MF8VTQ9l4

Link para userguide das bibliotecas PHP/Firebase:
https://firebase-php.readthedocs.io/en/latest/index.html
